src/appleseed/foundation/math/population.h

```diff
@@ -57,7 +57,7 @@ class Population
     Population();                           // empty population
 
     // Insert a new value into the population.
-    void insert(const ValueType& val);
+    void insert(const ValueType& val, const size_t amount = 1);
```

Comment:
Rename `amount` to `count` and update the comment ("Insert `count` times the value `val` into the population.")

Reply:
Ok !